docs/data-model.md

```diff
@@ -117,27 +115,36 @@ Indexes are segmented into groups of columns called shards (previously known as
 
 Query operations run in parallel, and they are evenly distributed across a cluster via a consistent hash algorithm.
 
-### View
+### Field Type
 
-Views represent the various data layouts within a Field. The primary View is called Standard, and it contains the typical Row and Column data. Time-based Views are automatically generated for each time quantum. Views are internally managed by Pilosa, and never exposed directly via the API.
+Upon creation, fields are configured to be of a certain type. Pilosa supports three field types: `set`, `int`, and `time`.
 
-#### Standard
+#### Set
 
-The standard View contains the same Row/Column format as the input data. 
+Set is the default field type in Pilosa. Set fields represent a standard, binary matrix of rows and columns where each row key represents a possible field value. The following example creates a `set` field called "info" with a ranked cache containing up to 100,000 records.
```

Comment:
I realize now that it might be confusing to see "Set" as both a field type and a query keyword. We should make sure the wording is unambiguous, at least in introductory docs pages